lib/stm32/l4plus/pwr.c

```diff
@@ -0,0 +1,85 @@
+/** @defgroup pwr_file PWR peripheral API
+ *
+ * @ingroup peripheral_apis
+ *
+ * @brief <b>libopencm3 STM32L4xx Power Control</b>
+ *
+ * @version 1.0.0
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2016 Benjamin Levine <benjamin@jesco.karoo.co.uk>
```

Comment:
If this file is benjamin's from 2016, shouldn't it all be shared?  are the functions here actually all new work just for l4plus?  In that case you probably _do_ want to change dates and copyrights here.

Reply:
pwr_enable_backup_domain_write_protect and pwr_disable_backup_domain_write_protect are from Benjamin, pwr_set_vos_scale is based on his version but ST added scale 1 boost on top of scale 1 and scale 2.

Reply:
so shouldn't those two functions go into the shared file then?